driver/prepare.go

```diff
@@ -123,6 +123,17 @@ func prepareContainer(cfg *drivers.TaskConfig, taskCfg TaskConfig) (syexec, erro
 		se.argvMountReadOnly = argvMountReadOnly
 	}
 
+    // set attributes
+	if len(taskCfg.Attributes) > 0 {
+		for _, attr := range taskCfg.Attributes {
+			split := strings.Split(attr, ":")
+			attribute := split[0]
+			value := split[1]
+			command := "set-attribute -p " + potName + " -A " + attribute + " -V " + value
```

Comment:
This should contain quoting, so things like `attributes = ["; shutdown -p now:val"]` won't be possible.